src/elasticsearch/addl_index_transformations/portal/add_partonomy.py

```diff
@@ -2,6 +2,30 @@
 from pathlib import Path
 from json import loads
 
+ORGAN_IRI = {
+    'BL': 'http://purl.obolibrary.org/obo/UBERON_0001255',
+    'RK': 'http://purl.obolibrary.org/obo/UBERON_0004539',
+    'LK': 'http://purl.obolibrary.org/obo/UBERON_0004538',
+    'HT': 'http://purl.obolibrary.org/obo/UBERON_0000948',
+    'LI': 'http://purl.obolibrary.org/obo/UBERON_0001155',
+    'LL': 'http://purl.obolibrary.org/obo/UBERON_0002168',
+    'RL': 'http://purl.obolibrary.org/obo/UBERON_0002167',
+    'LY': 'http://purl.obolibrary.org/obo/UBERON_0000029',
+    'SP': 'http://purl.obolibrary.org/obo/UBERON_0002106',
+    'TH': 'http://purl.obolibrary.org/obo/UBERON_0002370',
+    'UR': 'http://purl.obolibrary.org/obo/UBERON_0000056',
+    'LV': 'http://purl.obolibrary.org/obo/UBERON_0002107',
+    'OT': 'http://purl.obolibrary.org/obo/UBERON_0013702'
+}
+
+
+def get_organ_iri(doc):
+    if 'origin_sample' in doc and 'organ' in doc['origin_sample']:
+        organ_code = doc['origin_sample']['organ']
+        if organ_code in ORGAN_IRI:
+            return ORGAN_IRI[organ_code]
+    return None
```

Comment:
We already have code and configuration for mapping Elasticsearch organ terms to IRIs in [`portal-ui/organ-utils`](https://github.com/hubmapconsortium/portal-ui/blob/main/organ-utils/make_organ_dir.py#L192-L200) -- I don't think this logic should be repeated across the codebase.